<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>626</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Qtombo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/qtombo.qrc">
    <normaloff>:/icon/tombo.ico</normaloff>:/icon/tombo.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTomboTreeView" name="treeView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="indentation">
        <number>10</number>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>500</number>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>500</number>
       </attribute>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Verdana'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="tabStopWidth">
        <number>40</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionRename"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Root"/>
   <addaction name="separator"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normalon>:/icon/document-save.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSet_Root">
   <property name="icon">
    <iconset resource="../res/qtombo.qrc">
     <normaloff>:/icon/document-properties.svg</normaloff>:/icon/document-properties.svg</iconset>
   </property>
   <property name="text">
    <string>Set Root</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="../res/qtombo.qrc">
     <normaloff>:/icon/document-new.svg</normaloff>:/icon/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../res/qtombo.qrc">
     <normaloff>:/icon/edit-delete.svg</normaloff>:/icon/edit-delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete File</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRename">
   <property name="icon">
    <iconset resource="../res/qtombo.qrc">
     <normaloff>:/icon/Rename_icon.svg</normaloff>:/icon/Rename_icon.svg</iconset>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
   <property name="toolTip">
    <string>Rename</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QTomboTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">qtombotreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/qtombo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>treeView</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>textEdit</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSet_Root</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>setroot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>newfile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>deletefile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRename</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>rename()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>267</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>save()</slot>
  <slot>setroot()</slot>
  <slot>newfile()</slot>
  <slot>deletefile()</slot>
  <slot>rename()</slot>
 </slots>
</ui>
